src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1311,22 +1311,15 @@ protected override void Validate(object arguments, EngineIntrinsics engineIntrin
 
             if (MinLength == MaxLength && len != MaxLength)
             {
-                throw new ValidationMetadataException("ValidateCountNotExactlyEqual",
-                    null, Metadata.ValidateCountFailure,
-                    MinLength, MaxLength, len);
-            }
-
-            if (len < MinLength)
-            {
-                throw new ValidationMetadataException("ValidateCountSmallerThanMin",
-                    null, Metadata.ValidateCountFailure,
-                    MinLength, MaxLength, len);
+                throw new ValidationMetadataException("ValidateCountExactlyFailure",
```

Comment:
LGTM overall, but I suggest removing the "ly": `ValidateCountExactlyFailure` -> `ValidateCountExactFailure`